test/performance/ferguson/remote-array-write-access.chpl

```diff
@@ -17,4 +17,4 @@ stop();
 writeln(A[1]);
 writeln(A[n]);
 
-report(maxGets=1, maxOns=1);
+report(maxGets=2, maxOns=1, maxGetsToAdjust=1);
```

Comment:
I don't follow the `maxGetsToAdjust` logic (not saying it's wrong, I just don't grok it). Left to my own devices, I would just change this test to do something like:

```chpl
var maxGets = 1;
// accept higher comm counts on m1 mac for now -- see https://github.com/Cray/chapel-private/issues/3728
if CHPL_TARGET_PLATFORM == "darwin" && CHPL_TARGET_ARCH == "arm64" then maxGets = 2;
report(maxGets=maxGets, maxOns=1, maxGetsToAdjust=1);
```

to make it more obvious that it's just for arm-based macs we're seeing this, but maybe there's a benefit of the `maxGetsToAdjust` approach I don't understand.